app/assets/javascripts/app/views/help_view.js

```diff
@@ -187,8 +197,31 @@ app.views.Help = app.views.StaticContentView.extend({
     e.preventDefault();
   },
 
+  chat: function(e){
+    this.renderStaticSection("chat", "faq_chat", this.CHAT_SUBS);
+    this.menuClicked(e);
+
+    e.preventDefault();
+  },
+
   linkHtml: function(url, text) {
     return "<a href=\"" + url + "\" target=\"_blank\">" + text + "</a>";
+  },
+
+  linkImage: function(src, alt){
+    return '<div class="help-image"><img src="' + ImagePaths.get(src) + '" alt="' + alt + '"/></div>';
+  },
```

Comment:
Because with that argument we would still maintain about 6-10kloc more code in this repository. Please remove it.